index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio Project</title>
+    <script
+      src="https://kit.fontawesome.com/1572c8e00b.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="toolbar">
+      <div class="mylogo">
+        <a href="index.html">Juma</a>
+      </div>
+      <nav class="nav">
+        <ul>
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About Me</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </nav>
+      <div class="toggle" id="toggle">
+        <div class="menu"></div>
+      </div>
+    </header>
+    <sectiion class="headline">
```

Comment:
[OPTIONAL] Well done on getting the project approved 👍🏽 but just one thing I wanted to mention, although this doesn't change the way the project works but make sure to fix this typo, I believe you wanted to write `section` here. 😉